install/install.qmd

```diff
@@ -283,27 +284,71 @@ Common Points of failure; how to address them.
 
 See CmdStan Guide section [Troubleshooting the Installation](https://mc-stan.org/docs/cmdstan-guide/installation.html#troubleshooting-the-installation){target="_blank"} for further details.
 
-## Introductory Notebooks, Vignettes, and Tutorials
-
-### Python
-
-* CmdStanPy User's Guide Examples
+## High-level Stan Interfaces
```

Comment:
I like the idea of this, but it's probably worth noting somewhere in the table which of the above interfaces you need to install to use these (I think all 3 of the R options here can use _either_ RStan or CmdStanR)

Reply:
RStanArm doesn't require Rstan - it does require rstantools - https://cran.r-project.org/web/packages/rstanarm/index.html    that's the selling point for RStanArm - it installs easily from CRAN.